File: backend/src/main/java/com/demo/student_management/dto/admin/SystemOverviewResponse.java
package com.demo.student_management.dto.admin;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class SystemOverviewResponse {
    private long totalStudents;
    private long totalClasses;
    private long totalTeachers;
    private long totalSubjects;
    private List<PhanCongResponse> assignments;
    private List<GiaoVienResponse> teachers;
}

File: frontend/StudentManagement.Shared/Dtos/Admin/SystemOverviewResponse.cs
using System.Collections.Generic;

namespace StudentManagement.Shared.Dtos.Admin;

public sealed class SystemOverviewResponse
{
    public long TotalStudents { get; set; }
    public long TotalClasses { get; set; }
    public long TotalTeachers { get; set; }
    public long TotalSubjects { get; set; }
    public List<PhanCongResponse> Assignments { get; set; } = new();
    public List<GiaoVienResponse> Teachers { get; set; } = new();
}
